test/box/gh-5422-broken_snapshot.test.lua

```diff
@@ -74,7 +73,7 @@ test_run:cmd("switch default")
 os.execute(string.format('cp %s.save %s', snapfile, snapfile))
 -- write garbage at the middle of file
 file = io.open(snapfile, "r+b")
-file:seek("set", size)
+_ = file:seek("set", size / 2)
```

Comment:
You doubled data in snap and meanwhile halved offset of position to insert corrupted data. Seems that nothing should have changed...Also that's why using constants even in the test is mediocre idea: you could reduce your diff up to one line setting another constant for data size.

Reply:
> There's a lot of diff in the test. Does it worth the effort? Does it fix something? What identical cases are?

About identical test case, add some information to commit message - 
(in previous version we write different count of garbage in the middle
of snapshot, it's meaningless, because result is almost the same, only
the amount of data that can be read from snapshot differs).
Also i think that new test version more clear and readable